.circleci/config.yml

```diff
@@ -250,14 +250,13 @@ jobs:
 
   macos-integration-test:
     description: "Run integration tests for Flutter"
-    resource_class: macos.x86.medium.gen2
+    resource_class: macos.m1.large.gen1
     macos:
-      xcode: 13.4.1
+      xcode: 14.2.0
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
-      - setup-flutter:
-          version: '3.3.10'
```

Comment:
Hmm if I'm not wrong, by removing this, we will be pointing to the latest `main` in flutter? But seems to be what we do for the other jobs, so this might be ok.